### DISCUSSION TITLE
[Question]: How to configure Herd to  run a php 7.4 project

### DISCUSSION THREAD

**Discussion by MouhamedMokkhtar** [beyondcode/herd-community#339] [Q&A]
Operating system version
macOs Ventura 13.4.1

System architecture
ARM64 (M1, M2, etc)

Herd Version
1.4.0 (Build 20)

PHP Version
PHP 7.4

Error description
1) the file : " find-usable-php.php"  have a $minimumPhpVersion = '8.0';?
2) the phpMyAdmin.test shows error 

Rq : i put 2 errors because i think this 2 errors are related 

1)
im trying to run a laravel project with php 7.4 
i didn't have any environment for running laravel project before in my mac 
so the first thing i did is installing Herd and follow the documentation 
i found that i need to change the "find-usable-php.php" file always  to be able to run project with php 7.4

2)
the problem i have in the phpMyadmin.test site 
i added the phpMyAdmin folder to the Herd folder 
note that i dont have valet installed or any other configuration to run laravel before
the  phpMyAdmin.test site have this error :

Warning in ./../../../../../../../../Applications/Herd.app/Contents/Resources/valet/server.php#118
 array_merge(): Expected parameter 1 to be an array, null given

Backtrace

./../../../../../../../../Applications/Herd.app/Contents/Resources/valet/server.php#118: array_merge(
NULL,
array,
)
{closure}()

Relevant log output
No response

**Comment by mpociot** [comment]
Please create an file at `~/Library/Application Support/Herd/config/herd.json` containing just `{}`
This issue seems to be PHP 7.4 specific, creating the file manually fixes it.
